### ISSUE TITLE
[feature request] list records by specified range

### ISSUE THREAD

**Issue by wh75er** [dominikbraun/timetrace#222]
At the end of the week I like to make a summary on my project[s]. It would be handy to list records not only for a specific day, but for specified time range. I haven't found how to do it in the current version 

---

### Usage example

`timetrace list records 0000-00-00 9999-99-99 -p foo` 
will output all my records and total time spent in time range [0000-00-00, 9999-99-99] for foo project

### Implementation 

Not ready yet. It looks like necessary backbone feature to me. But probably could be implemented as an extension (with plugins)

Basically, I think `timetrace list records` must take 2 arguments, instead of 1. Second argument is optional - `start_time [end_time]`
Then we need to iterate over record_dirs and records, which matches specified boundaries

---

I want to work on this feature in the near future, but first I'd like to know your thoughts on it, @dominikbraun. Is it possible to add it to the core API or should it be implemented as a plugin

**Comment by dominikbraun** [comment]
Hi @wh75er, thanks for you feature request! I think this can be implemented in the timetrace core.

**Comment by wh75er** [comment]
Well, I've made a research to implement this feature and found this one. I can list all records for the specified time range with

`timetrace report --start 0000-00-00 --end 9999-99-99`

This will list every information I needed. I think this issue can be closed
